src/components/FlowRunCreateForm.vue

```diff
@@ -151,6 +168,13 @@
 
     emit('submit', resolvedValues)
   })
+
+  watch(addRetries, value => {
+    if (!value) {
+      retries.value = null
+      retryDelay.value = null
+    }
+  })
```

Comment:
I wonder if you could just clear the retries, if they have a retry delay on submit just toss it out. That way if I'm fiddling with the retries it doesn't clear my delay I might want to keep. nbd

Reply:
Removing the toggle means this isn't needed anymore. Simpler might be better

Reply:
The code would be simpler but I think the UX is better with the toggle. What do you guys think?

Reply:
Below it you have the "Start [now | later]" toggle, which I assume shows more form if you select "later", the toggle feels a part of that theme.

It is a little strange to opt in, then have optional fields, at that point those fields are required for the toggle to be truthful. I also wonder if it should be separated out like "Start" is, instead of an "Add Retries" label, make a "Retry on Fail" (or something) header in it's own section. I like what Craig mentioned about making these fields align horizontally, since they both accept a pretty short number.

The Retries and Start parts of the form might be a bit pre-optimized, idk (not sure how much form is behind the Start toggle). I don't feel strongly, but if you made me pick, I'd probably roll with continuing reveal pattern with the design nudges noted above here.

Reply:
The start now/later is a choice the user has to make. If they pick later there are required fields to fill in. 

This toggle is purely for display purposes (the form acts the same with or without it) which is why I'd vote to remove it. 

But I don't feel strongly.